### ISSUE TITLE
Rambox gets too hot on MacOS

### ISSUE THREAD

**Issue by alfonsotesone** [ramboxapp/community-edition#2721]
# Actual behavior

When I open Rambox many processes called "Rambox Helper" are created and the temperature of my Mac reaches 80/90 °. Activity tracking currently tells me 37 threads have been created. 😔

### ENVIRONMENT

> Rambox: v0.7.5

> OS: MacOS

> Arch: x64

**Comment by fvulich** [comment]
It seems that an app inside Rambox CE is crashing, could you try to do this? Once you notice that Rambox CE is consuming an abnormal amount of resources, try to disable apps one by one to see if one of them is the culprit.

If that doesn't do the trick, could you try doing a Clear cache & a Clear Storage and tell us how it goes? Before doing this, backup all your services following [this tutorial](https://github.com/ramboxapp/community-edition/wiki/Backup-Rambox-CE-services) , because you will lose them.
![image](https://user-images.githubusercontent.com/46904390/80006611-8090ec00-849b-11ea-9af6-cd8c75d01d2e.png)


If that doesn't work either, please, try uninstalling/reinstalling Rambox CE, [this guide (Clean uninstall)](https://github.com/ramboxapp/community-edition/wiki/Clean-Uninstall) will help you out.

**Comment by alfonsotesone** [comment]
Here I am.
I made a first attempt, reducing the number of sessions open on rambox a little at a time (Whatsapp, Telegram, Gmail, etc). But the situation has not improved, the temperature continued to remain high and even on the activity monitor it had 42/44 active threads. I have performed a cache and local storage clean but also without success.

In the end I uninstalled Rambox (removing all the folder from memory with the terminal) and reinstalled by configuring again all the sessions that I used even before. The situation seems slightly improved, the temperature is now around 50 ° but the active threads are still high (around 30/34).

Screen before deleting Rambox:
<img width="556" alt="Schermata 2020-05-29 alle 16 08 37" src="https://user-images.githubusercontent.com/20597965/83271423-2685f380-a1ca-11ea-816c-4ac2f086c679.png">

<img width="275" alt="Schermata 2020-05-29 alle 16 16 17" src="https://user-images.githubusercontent.com/20597965/83271494-3c93b400-a1ca-11ea-9c4b-3465e0ad26cf.png">


Screen after new installation:
<img width="550" alt="Schermata 2020-05-29 alle 16 35 12" src="https://user-images.githubusercontent.com/20597965/83271802-a14f0e80-a1ca-11ea-8101-723a56c71dac.png">

<img width="250" alt="Schermata 2020-05-29 alle 16 36 55" src="https://user-images.githubusercontent.com/20597965/83271811-a57b2c00-a1ca-11ea-9c68-a343cdc41e2a.png">
